src/Entity/Item.php

```diff
@@ -77,6 +77,38 @@ public function setId( $id ) {
 	}
 
 	/**
+	 * @param string $languageCode
+	 * @param string $value
+	 *
+	 * @return string
```

Comment:
You want to go back to 2012 in the Wikibase.git version history and see the first versions of Entity ;p We had some crazy stuff, such as putting Entity and EntityDocument in one class, and having a database query in the constructor of SiteLink.